lib/openstudio/extension/runner_config.rb

```diff
@@ -40,17 +42,31 @@ def add_config_option(name, value)
         @data[name.to_sym] = value
       end
 
+      def self.get_local_bundle_config_path(dirname)
+        # Bundler.settings reads from pwd, and I can't test with this...
+        bundler_settings = Bundler::Settings.new(File.join(dirname, '.bundle'))
+
+        if (bundle_install_path = bundler_settings.path.explicit_path)
+          puts "Defaulting bundle_install_path to bundle's local config value: '#{bundle_install_path}'"
+        else
+          puts "Defaulting to .bundle/install (and ignoring system wide: Bundler.configured_bundle_path.base_path=#{bundler_settings.path.base_path})"
+          bundle_install_path = File.join(dirname, '.bundle/install/')
+        end
+
+        return bundle_install_path
+      end
+
       ##
       # Return the default runner configuration
-      def self.default_config
+      def self.default_config(dirname)
         return {
           file_version: '0.1.0',
           max_datapoints: 1E9.to_i,
-          num_parallel: 2,
+          num_parallel: Parallel.processor_count,
           run_simulations: true,
           verbose: false,
           gemfile_path: '',
-          bundle_install_path: ''
+          bundle_install_path: get_local_bundle_config_path(dirname),
```

Comment:
use that + default num_parallel to available (nproc)

Reply:
maybe `processor_count - 1` ?

Reply:
Meh, I understand the rationale. But I really doubt the ruby thread is going to use resources while the rest is doing what it has to do (it's blocking if I reminder correctly). While on my machine with 16 threads I'd be totally ok with a spare thread, on GHA where you have 2 or 3 total this might be a waste?

Reply:
I went ahead and did it in fbc2936bcc38e6659972a34cc3ece3677fa328dc